decidim-core/lib/decidim/encryptor.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Encryptor
```

Comment:
This module's methods (name and documentation) are tightly coupled to newsletters. Since this is a top-level interface (Decidim::Encryptor) we either find a more general case, or this should live inside code specific to newsletters. It could live into the controller or maybe inside a more specific `Decidim::NewsletterEncryptor` module/class/ Otherwise, we're leaking abstractions.